src/main/java/seedu/address/model/module/Module.java

```diff
@@ -17,6 +19,7 @@
 public class Module {
     private final ModuleName name;
     private final ZoomLink zoomLink;
```

Comment:
Perhaps this field could be removed since we are storing multiple zoom links in the hashmap?

Reply:
Yup, I will remove it. Just want to make sure the HashMap implementation is correct before deleting.